tool/tsh/tsh.go

```diff
@@ -689,7 +689,10 @@ func Run(args []string, opts ...cliOption) error {
 	setEnvFlags(&cf, os.Getenv)
 
 	confOptions, err := loadConfig(cf.HomePath)
-	if err != nil && !trace.IsNotFound(err) {
+	if trace.IsNotFound(err) {
+		err = nil
+	}
```

Comment:
Hm, I wonder if it would be a little cleaner interface if we made `loadConfig` return an empty config in case the config file does not exist. Then the caller won't have to worry about handling the not found error. cc @lxea

Reply:
That makes sense to me